source/content/overview/matrix.md

```diff
@@ -0,0 +1,56 @@
+# Matrix construction
+
+We normally use Brightway to do life cycle assessment calculations using matrices. This section describes how matrices are constructed. In a standard calculation, we would have three matrices:
+
+* `technosphere` matrix: Our model of the supply chain. It has columns of processes, and rows of products. Chimaera process+product nodes will have both a row and a column.
```

Comment:
...glossary again.

Reply:
OK, I am not as convinced on this point but will try to make it fit naturally.